bootloader/src/pyi_win32_utils.c

```diff
@@ -435,6 +435,55 @@ pyi_win32_utf8_to_mbs(char * dst, const char * src, size_t max)
     }
 }
 
+
+/* Retrieve the SID of the current user.
+ *  Returns SID string on success, NULL on failure.
+ */
+static char *
+pyi_win32_get_user_sid()
+{
+    HANDLE process_token;
+    DWORD user_info_size, result;
+    PTOKEN_USER user_info;
+    wchar_t *wsid = NULL;
+    char *sid;
+
+    // Get access token for the calling process
+    if (!OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &process_token)) {
+        return NULL;
+    }
+
```

Comment:
Please remove these blank lines. The block is already visually separated by having a line with only the closing bracket. Adding a blank line visually breaks the code.

Reply:
Done.